app/views/users/sessions/new.html.erb

```diff
@@ -0,0 +1,17 @@
+<header>Log in</header>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+
+  <%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder: "Email" %>
+  <%= f.password_field :password, autocomplete: "current-password", placeholder: "Password" %>
+
+  <div class="greenButton">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+
+<div class="headerBack">
+  <%= link_to "<", users_path	 %>
+</div>
+
+<%= render "users/shared/links" %>
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/219305575-637d2d04-8101-4a44-a5f0-1362b8a73eb0.png)

**[Optional]**

- [ ] You have a horizontal scroll on your login page which is not professional and following the design at all :thinking: Kindly fix this issue and follow the design guidelines to make your project look professional :+1:
- [ ] Sign up and Forgot your password? options should be in the center of the screen instead of being at the left side of the screen :thinking: Kindly follow the design to make your project look professional :+1: